serde_with/src/de/impls.rs

```diff
@@ -33,6 +37,11 @@ macro_rules! foreach_map_create {
             HashMap<K: Eq + Hash, V, S: BuildHasher + Default>,
             (|size| HashMap::with_capacity_and_hasher(size, Default::default()))
         );
+        #[cfg(feature = "hashbrown")]
+        $m!(
+            HashbrownMap<K: Eq + Hash, V, S: BuildHasher + Default>,
+            (|size| HashbrownMap::with_capacity_and_hasher(size, Default::default()))
+        );
```

Comment:
Flagging that I followed the `indexmap` template but I have not properly setup the dependencies to enable hashbrown in a `no_std` environment (assuming that's even possible). Rather than take a punt I thought I would wait for your guidance on the matter @jonasbb

Reply:
I see nothing that would prevent the `no_std` usecase. hashbrown does not even have a `std` feature, probably because it is so deeply embedded in the Rust standard library.

You can change this line in the CI tests to include the `hashbrown_0_14` feature. That will compile the code on a target without std support.
https://github.com/jonasbb/serde_with/blob/fc30b63125c501b12e1c9e53e50a7618c4180b47/.github/workflows/ci.yaml#L92C157-L92C157